config/games/4inarow.ts

```diff
@@ -0,0 +1,40 @@
+import { ServerGame } from "../../types/types";
+import { postJson } from "../../utils/utils";
+
+const game: ServerGame = {
+    id: "4inarow",
+    name: "4 in a Row",
+    author: "fravic",
+    basedOn: {
+        game: "Connect Four",
```

Comment:
We can supply more “based on” information:

```suggestion
        game: "Connect Four",
        link: "https://en.wikipedia.org/wiki/Connect_Four",
        bggId: 2719,
```

Reply:
Thanks, updated!